src/articles/articles.service.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { ArticleDto } from './article.dto';
+import { Article } from './article.entity';
+
+@Injectable()
+export class ArticlesService {
+  constructor(
+    @InjectRepository(Article)
+    private articleRepository: Repository<Article>,
+  ) {}
+
+  async createArticle(articleDto: ArticleDto): Promise<Article> {
+    const article = new Article();
+    article.title = articleDto.title;
```

Comment:
How do you think, compared to other methods of the class, could we refactor this?

Reply:
@haltaction I've created a separate method for this

Reply:
Great :+1: